src/test/java/Bwead/BweadTest.java

```diff
@@ -0,0 +1,107 @@
+package Bwead;
```

Comment:
this package seems too packed rn 🤔 

Suggestion: Can separate files according to their uses into different directories of the package like Bwead.main, Bwead.tasks, Bwead.tests, etc.